pages/doc/csp_invite-AoA-users_tutorial.md

```diff
@@ -22,15 +22,20 @@ To invite users, you must have the VMware Cloud **Organization Owner** or **Orga
 
 ## Roles to Assign
 
-When you invite new users, you must assign them:
+To invite new users, you assign them:
 
-* A role within the VMware Cloud organization, such as **Organization Administrator**, **Organization Owner**, or **Organization Member**. 
+* A role within the VMware Cloud organization, such as **Organization Administrator**, **Organization Owner**, or **Organization Member**. See [What organization roles are available in VMware Cloud Services](https://docs.vmware.com/en/VMware-Cloud-services/services/Using-VMware-Cloud-Services/GUID-C11D3AAC-267C-4F16-A0E3-3EDF286EBE53.html) in the VMware Cloud services documentation.
     
   Note that you can assign the **Organization Owner** role to another user only if you have the **Organization Owner** role.
 
-* A role within the Operations for Applications service instance. 
+* A role within the Operations for Applications service instance. We provide a number of [Operations for Applications service roles](csp_users_roles.html#operations-for-applications-service-roles-built-in).
+
+   Note that in a multi-tenant Operations for Applications environment, you must specify the service instance (tenant) for which you want to assign the service role. You can assign different service roles for different service instances (tenants). You invite the users only to the tenants for which you assigned them service roles.
+
+* Optionally, a custom role created in the VMware Cloud organization. [Custom roles](csp_users_roles.html#create-edit-or-delete-a-custom-role) are composed of different service permissions.
+
+   Note that a custom role with an Operations for Applications permission applies only if the user has at least one Operations for Applications service role. In a multi-tenant Operations for Applications environment, custom roles apply to all service instances (tenants) for which the user has at least one Operations for Applications service role.
```

Comment:
You invite the users only to the tenants for which you assigned them service roles.  ----> You invite the users only to the tenants that have the assigned service roles.

Reply:
I think it's okay now, because you don't assign the service roles to the tenants.